support/mock/mockrt.go

```diff
@@ -183,6 +184,9 @@ func (rt *Runtime) Send(toAddr addr.Address, methodNum abi.MethodNum, params run
 	if len(rt.expectSends) == 0 {
 		rt.t.Fatalf("unexpected expectedMessage to: %v method: %v, value: %v, params: %v", toAddr, methodNum, value, params)
 	}
+	if params == nil {
+		params = adt.EmptyValue{}
+	}
```

Comment:
Unsure if this is necessary. Could we just use nil?

Reply:
Right, on further thought I think we should normalize to nil.